File: Data Structures & Algorithms/missing-number/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    missingNumber(nums) {
        let n = nums.length
        const sum = Math.floor(n*(n+1)/2)
        let accured = 0
        for (const num of nums) {
            accured += num
        }
        return sum - accured
    }
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i,k = 0,0
        while i<len(nums) and k<len(nums):
            if i==0 or nums[i]>nums[i-1]:
                nums[k] = nums[i]
                i+=1
                k+=1
            else:
                i+=1
        nums = nums[:k]
        return len(nums)

